scripts/superdesk-authoring/authoring/controllers/ChangeImageController.js

```diff
@@ -1,5 +1,7 @@
-ChangeImageController.$inject = ['$scope', 'gettext', 'notify', 'modal', '$q', 'lodash', 'api', '$rootScope', 'config'];
-export function ChangeImageController($scope, gettext, notify, modal, $q, _, api, $rootScope, config) {
+ChangeImageController.$inject = ['$scope', 'gettext', 'notify', 'modal', '$q', 'lodash', 'api', '$rootScope', 'config',
+                                 'authoringWorkspace', 'archiveService'];
+export function ChangeImageController($scope, gettext, notify, modal, $q, _, api, $rootScope, config, authoringWorkspace,
+                                      archiveService) {
```

Comment:
better to return the line at the same position you did for `ChangeImageController.$inject` :trollface: